use https rather than http for download URLs, to fix broken tests;
see https://github.com/mdtraj/mdtraj/issues/2126 and https://github.com/mdtraj/mdtraj/pull/2106
diff -ru mdtraj-1.9.9.orig/tests/test_dssp.py mdtraj-1.9.9/tests/test_dssp.py
--- mdtraj-1.9.9.orig/tests/test_dssp.py	2022-10-27 17:00:03.000000000 +0200
+++ mdtraj-1.9.9/tests/test_dssp.py	2026-01-21 07:54:50.601558615 +0100
@@ -61,7 +61,7 @@
     # 1COY gives a small error, due to a broken chain.
     pdbids = ['1GAI', '6gsv', '2AAC']
     for pdbid in pdbids:
-        t = md.load_pdb('http://www.rcsb.org/pdb/files/%s.pdb' % pdbid)
+        t = md.load_pdb('https://www.rcsb.org/pdb/files/%s.pdb' % pdbid)
         t = t.atom_slice(t.top.select_atom_indices('minimal'))
         assert_(call_dssp(tmpdir, t), md.compute_dssp(t, simplified=False)[0])
 
diff -ru mdtraj-1.9.9.orig/tests/test_pdb.py mdtraj-1.9.9/tests/test_pdb.py
--- mdtraj-1.9.9.orig/tests/test_pdb.py	2022-11-06 02:55:17.000000000 +0100
+++ mdtraj-1.9.9/tests/test_pdb.py	2026-01-21 07:54:50.601558615 +0100
@@ -195,8 +195,8 @@
 
 def test_pdb_from_url():
     # load pdb from URL
-    t1 = load_pdb('http://www.rcsb.org/pdb/files/4ZUO.pdb.gz')
-    t2 = load_pdb('http://www.rcsb.org/pdb/files/4ZUO.pdb')
+    t1 = load_pdb('https://www.rcsb.org/pdb/files/4ZUO.pdb.gz')
+    t2 = load_pdb('https://www.rcsb.org/pdb/files/4ZUO.pdb')
     eq(t1.n_frames, 1)
     eq(t2.n_frames, 1)
     eq(t1.n_atoms, 6200)
@@ -238,8 +238,8 @@
     t1 = load_pdb(get_fn('1ncw.pdb.gz'))
 
 def test_1vii_url_and_gz(get_fn):
-    t1 = load_pdb('http://www.rcsb.org/pdb/files/1vii.pdb.gz')
-    t2 = load_pdb('http://www.rcsb.org/pdb/files/1vii.pdb')
+    t1 = load_pdb('https://www.rcsb.org/pdb/files/1vii.pdb.gz')
+    t2 = load_pdb('https://www.rcsb.org/pdb/files/1vii.pdb')
     t3 = load_pdb(get_fn('1vii.pdb.gz'))
     t4 = load_pdb(get_fn('1vii.pdb'))
     eq(t1.n_frames, 1)
